templates/dashboard.html

```diff
@@ -4,7 +4,6 @@
 
 {% block content %}
 
-<div class="container">
```

Comment:
I noticed all of your code is wrapped in a parent `div.container`. So it would be  more useful to do this in the base.